### PR TITLE
Update Readme.md

### PR DESCRIPTION
**Pull request by troydai**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,13 +27,13 @@ $ ./build.sh
...
+You cand find the output under `artifacts/publish/<runtime id>/<target framework>/`.
 
-### VS Code
+The executable is `OmniSharp.exe` or `OmniSharp`.
 
-Add following setting to your User Settings or Workspace Settings.
+### VS Code
 
-_Update path to fit your environment_
+Add following setting to your [User Settings or Workspace Settings](https://code.visualstudio.com/Docs/customization/userandworkspace).
```
Comment (willl) [inline README.md]: "Add _the_ following"

Comment (troydai) [inline README.md]: yup.

File: README.md [hunk 2]
```diff
@@ -27,13 +27,13 @@ $ ./build.sh
 
 ```
 
-You cand find the output under `artifacts/publish/<target framework>/`. The executable is `OmniSharp.exe` or `OmniSharp`.
+You cand find the output under `artifacts/publish/<runtime id>/<target framework>/`.
 
-### VS Code
+The executable is `OmniSharp.exe` or `OmniSharp`.
```
Comment (willl) [inline README.md]: Add the word _either_ between _executable_ and _is_

File: README.md [hunk 3]
```diff
@@ -27,13 +27,13 @@ $ ./build.sh
 
 ```
 
-You cand find the output under `artifacts/publish/<target framework>/`. The executable is `OmniSharp.exe` or `OmniSharp`.
+You cand find the output under `artifacts/publish/<runtime id>/<target framework>/`.
```
Comment (willl) [inline README.md]: can instead of cand. There's an extra d.

File: README.md (hunk 2 above)
Comment (troydai) [inline README.md]: yup

File: README.md (hunk 3 above)
Comment (troydai) [inline README.md]: yup

**Pull request merged**